sources/packages/web/src/services/http/dto/ApplicationRestrictionBypass.dto.ts

```diff
@@ -0,0 +1,33 @@
+import { RestrictionType } from "@/types";
+
+/**
+ * DTO class for AEST application restriction bypass summary.
+ */
+export class ApplicationRestrictionBypassSummary {
+  id: number;
+  restrictionType: RestrictionType;
+  restrictionCode: string;
+  isActive: boolean;
+  isBypassActive: boolean;
+}
+
+/**
+ * DTO class for AEST application restriction bypass history.
+ */
+export class ApplicationRestrictionBypassHistoryAPIOutDTO {
+  bypasses: ApplicationRestrictionBypassSummary[];
+}
+
+/**
+ * Indicates if the bypassed restriction is active or removed.
+ */
+export enum ApplicationRestrictionBypassStatus {
```

Comment:
This is a status chip label type for `ApplicationRestrictionBypass`. 

Please move this type outside DTO and into the `types/contract/StatusChip.ts` and suggested to name the object as `ApplicationRestrictionBypassStatusChipLabelTypes` or `BypassStatusChipLabelTypes`

Reply:
Updated.